DevOps/LICENSE

```diff
@@ -0,0 +1,201 @@
+   Apache License
```

Comment:
Including a license for our project is probably a good idea, but it should be for the whole project. Recommend removing it here.

Reply:
This applies to the DevOps_Team Eric Kniggie is part of the Front-End team